skillmap/src/lib/skillMapUtils.ts

```diff
@@ -160,9 +160,10 @@ export function isRewardNode(node: MapNode) {
     return node.kind === "reward" || node.kind === "completion";
 }
 
-export function getFlattenedHeaderIds(user: UserState, pageSource: string): string[] {
+export function getFlattenedHeaderIds(user: UserState, pageSource: string, existing: UserState): string[] {
```

Comment:
Does this only get the ids for unstarted projects in the cloud? should this be called getHeaderIdsForUnstartedMaps ?

Reply:
yeah, i guess "rename symbol" was trying to be clever and renamed the import but not the definition. Fixed!